.changeset/mighty-seas-pump.md

```diff
@@ -0,0 +1,2 @@
+---
+---
```

Comment:
No need to do this; this package is private. CI shouldn't fail here. I'll update the docs to be clearer.